site/src/xServices/users/usersXService.ts

```diff
@@ -348,6 +349,10 @@ export const usersMachine = createMachine(
       assignUpdateRolesError: assign({
         updateUserRolesError: (_, event) => event.data,
       }),
+      clearUsers: assign((context: UsersContext) => ({
+        ...context,
+        users: undefined,
+      })),
```

Comment:
So glad you fixed this. I'm not opposed to `clearUsers` but curious if we can ruse `assignUsers` and just overwrite the old users array with the new empty array.

Reply:
Sounds good to me!